internal/core/software_renderer.rs

```diff
@@ -1079,7 +1097,6 @@ impl<'a, T: ProcessScene> crate::item_rendering::ItemRenderer for SceneBuilder<'
             if border.get() > 0.01 as Coord {
                 // FIXME: radius
```

Comment:
```suggestion
```

If you apply my suggestion bellow, you no longer needs this if (because the border_color.apha test takes care of it)
Also, the radius was implemented long ago (above) so that comment has been outdated for a long time ;)